guillotina/api/content.py

```diff
@@ -194,7 +194,9 @@ class DefaultPOST(Service):
 
         # Create object
         try:
-            obj = await create_content_in_container(self.context, type_, new_id, **options)
+            obj = await create_content_in_container(
+                self.context, type_, new_id, skip_global_constrain=False, **options
```

Comment:
How about instead of `skip_global_constrain`, we just have a way to ignore all constraints?

So just rename to `check_constraint` just like the `check_security`